hint.css

```diff
@@ -0,0 +1,45 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.question-container {
+  width: 80%;
+  margin: auto;
+}
+
+.hint-container {
+  padding: 20px;
+  background-color: lightgoldenrodyellow;
+  border-radius: 8px;
+  justify-content: flex-start;
+}
+
+h1 {
```

Comment:
The `h1` tag is convenient, so it can be used for a simple site like this one. However, if the page becomes more complex, it is better to avoid using `h1` and to use more appropriate names instead.